modules/thermo-cycler/production/eepromWriter/eepromWriter.ino

```diff
@@ -0,0 +1,215 @@
+#include <Wire.h>
+
+#define EEPROM_ADDR       0x52
+
+#define MAX_IN_NUM_LEN    17  // max serial = 17 (TCV01yyyymmddAnnn) | max model = 3 //vnn
```

Comment:
See the similar comment on eeprom.h - this stuff should be in a shared data file or something